helium_miner_dashboard.json

```diff
@@ -502,19 +519,16 @@
       "points": false,
       "renderer": "flot",
       "seriesOverrides": [
-        {
```

Comment:
As I commented in the commit and PR, Grafana stripped all `hashKey` lines when I import then export your original.